inspirehep/modules/theme/views.py

```diff
@@ -344,6 +345,24 @@ def ping():
     return 'OK'
 
 
+@blueprint.route('/health')
+@time_execution
+def health():
```

Comment:
not sure this is the right place for it, as `theme` is the old ui that is being replaced by the new react one. @drjova ?

Reply:
Same for the above `ping`, which is used from nagios to check nginx status.
Shell we move them both somewhere else?

Reply:
And this is also the place where we serve the root '/'. So we will move them together, but not in this PR.